### PR TITLE
Metrics Operators Index

### PR DESCRIPTION
**Pull request by Melvin-CnC**
Made changes as per the Open Source Docs Migration Plan.

## Purpose of this pull request

This pull request (PR) to edit the Metrics Operators Index under Metrics.

Issue number: 

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by Melvin-CnC** [comment]
@kimsauce 
Seems like a Docusaurus issue. I'm troubleshooting it. Will ask Jagadish or Megha for help.

File: docs/metrics/metrics-operators/index.md [hunk 1]
```diff
@@ -9,14 +9,162 @@ This guide describes Sumo Logic metrics operators and provides usage examples.
...
+</div>
+<div className="box smallbox5 card">
+  <div className="container">
+  <a href="/docs/metrics/metrics-operators/count">  <img src={useBaseUrl('img/icons/metrics.png')} alt="icon" width="40"/><h4>count</h4></a>
+  <p>Learn how to count the total number of time series that match the query.</p>
+  </div>
+</div>
+<div className="box smallbox6 card">
+  <div className="container">
+  <a href="/docs/apm/real-user-monitoring/delta/"><img src={useBaseUrl('img/icons/metrics.png')} alt="icon" width="40"/><h4>delta</h4></a>
```
Comment (kimsauce) [inline docs/metrics/metrics-operators/index.md:49]:
```suggestion
  <a href="/docs/metrics/metrics-operators/delta/"><img src={useBaseUrl('img/icons/metrics.png')} alt="icon" width="40"/><h4>delta</h4></a>
```

File: docs/metrics/metrics-operators/index.md [hunk 2]
```diff
@@ -9,14 +9,162 @@ This guide describes Sumo Logic metrics operators and provides usage examples.
...
+  </div>
+  <div className="box smallbox15 card">
+    <div className="container">
+    <a href="/docs/metrics/metrics-operators/outlier"><img src={useBaseUrl('img/icons/metrics.png')} alt="icon" width="40"/><h4>outlier</h4></a>
+    <p>Learn how to identify metrics data points that are outside the range of expected values.</p>
+    </div>
+  </div>
+  <div className="box smallbox16 card">
+    <div className="container">
+    <a href="/docs/apm/real-user-monitoring/parse/"><img src={useBaseUrl('img/icons/metrics.png')} alt="icon" width="40"/><h4>parse</h4></a>
```
Comment (kimsauce) [inline docs/metrics/metrics-operators/index.md:109]:
```suggestion
    <a href="/docs/metrics/metrics-operators/parse/"><img src={useBaseUrl('img/icons/metrics.png')} alt="icon" width="40"/><h4>parse</h4></a>
```

File: docs/metrics/metrics-operators/index.md [hunk 3]
```diff
@@ -9,14 +9,162 @@ This guide describes Sumo Logic metrics operators and provides usage examples.
...
+</div>
+<div className="box smallbox10 card">
+  <div className="container">
+  <a href="/docs/metrics/metrics-operators/filter"><img src={useBaseUrl('img/icons/metrics.png')} alt="icon" width="40"/><h4>filter</h4></a>
+  <p>Learn how to limit the results returned by a metric query.</p>
+  </div>
+</div>
+  <div className="box smallbox11 card">
+    <div className="container">
+    <a href="/docs/metrics/metrics-operators/histogram_quantile"><img src={useBaseUrl('img/icons/metrics.png')} alt="icon" width="40"/><h4>histogram_quantile</h4></a>
```
Comment (kimsauce) [inline docs/metrics/metrics-operators/index.md:79]:
```suggestion
    <a href="/docs/metrics/metrics-operators/histogram-quantile"><img src={useBaseUrl('img/icons/metrics.png')} alt="icon" width="40"/><h4>histogram_quantile</h4></a>
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
broken links

File: docs/metrics/metrics-operators/index.md [hunk 4]
```diff
@@ -9,14 +9,162 @@ This guide describes Sumo Logic metrics operators and provides usage examples.
...
+</div>
+<div className="box smallbox4 card">
+  <div className="container">
+  <a href="/docs/metrics/metrics-operators/bottomk"><img src={useBaseUrl('img/icons/metrics.png')} alt="icon" width="40"/><h4>bottomk</h4></a>
+  <p>Learn how to apply a specified aggregation function to the time series that match the query selector.</p>
+  </div>
+</div>
+<div className="box smallbox5 card">
+  <div className="container">
+  <a href="/docs/metrics/metrics-operators/count">  <img src={useBaseUrl('img/icons/metrics.png')} alt="icon" width="40"/><h4>count</h4></a>
```
Comment (kimsauce) [inline docs/metrics/metrics-operators/index.md:43]:
```suggestion
  <a href="/docs/metrics/metrics-operators/count"><img src={useBaseUrl('img/icons/metrics.png')} alt="icon" width="40"/><h4>count</h4></a>
```

**Review by kimsauce — APPROVED** [review]

**Pull request merged**